access-tutorials/part-3-pocket-configuration.md

```diff
@@ -118,7 +118,7 @@ To create a new config file:
 1.  Run the following command, which will create the default `config.json` file, add the seeds, set port 8081 to 8082, and increase the RPC timeout value:
 
     ```bash
-    export SEEDS=$(curl -s https://raw.githubusercontent.com/pokt-network/pocket-seeds/main/mainnet.txt | head -n 4 | tr '\n' ',' | sed 's/,$//')
+    export SEEDS=$(curl -s https://raw.githubusercontent.com/pokt-network/pocket-seeds/main/mainnet.txt | head -n 6 | tr '\n' ',' | sed 's/,$//')
```

Comment:
How about merging two commands in one code section like this?

```shell
export SEEDS=$(curl -s https://raw.githubusercontent.com/pokt-network/pocket-seeds/main/mainnet.txt \
  | tr '\n' ',' \
  | sed 's/,*$//')
pocket util print-configs \
  | jq --arg seeds "$SEEDS" '.tendermint_config.P2P.Seeds = $seeds' \
  | jq '.pocket_config.rpc_timeout = 15000' \
  | jq '.pocket_config.rpc_port = "8082"' \
  | jq '.pocket_config.remote_cli_url = "http://localhost:8082"' \
  | jq . > ~/.pocket/config/config.json
```

I also added the following changes in the suggestion above:
- No need to limit the number of seeds.  Let's cut `head` command.
- Regex should be `s/,*$//` (with a asterik) to remove multiple commas at the end
- We should start with the default config by running `pocket util print-configs` instead of an empty object `{}`.
- Split lines with backquotes for easier reading

Reply:
Thanks! That was a very good feedback.
Done